py3status/modules/weather_yahoo.py

```diff
@@ -8,11 +8,34 @@
 Find your city code using:
     http://answers.yahoo.com/question/index?qid=20091216132708AAf7o0g
 
+Find your woeid using:
+    http://woeid.rosselliot.co.nz/
+
 Configuration parameters:
     - cache_timeout : how often to check for new forecasts
     - city_code : city code to use
     - forecast_days : how many forecast days you want shown
+    - forecast_format : possible placeholders: {icon}, {low}, {high}, {units},
+        {text}
+    - forecast_include_today: show today forecast as well, default false
```

Comment:
well I don't have the API JSON in mind so I guess I'll trust you on the fact that this makes sense :) mind updating the description about it a little more plz ?